### PR TITLE
Feature bundled specs command

### PR DESCRIPTION
**Pull request by f-arruza**

### CODE REVIEW

File: docs/command_line.md [hunk 1]
```diff
@@ -67,3 +69,28 @@ pyms encrypt 'mysql+mysqlconnector://important_user:****@localhost/my_schema'
...
+pyms merge-swagger [-h] [-f FILE]
+```
+
+```bash
+optional arguments:
+  -h, --help            show this help message and exit
+  -f FILE, --file FILE  Swagger file path
+```
+
+This command use [prance](https://github.com/jfinkhaeuser/prance) to validate the API specification and generate a single YAML file. Has an optional argument to indicate the main file path of the API specification.
```
Comment (alexppg) [inline docs/command_line.md:87]:
```suggestion
This command uses [prance](https://github.com/jfinkhaeuser/prance) to validate the API specification and generate a single YAML file. It has an optional argument to indicate the main file path of the API specification.
```

File: docs/command_line.md [hunk 2]
```diff
@@ -67,3 +69,28 @@ pyms encrypt 'mysql+mysqlconnector://important_user:****@localhost/my_schema'
...
+
+!!! warning
+    You must run first `pip install prance==0.18.2`
```
Comment (alexppg) [inline docs/command_line.md:90]: In this doc there's no mention of the additional dependencies that the commands might need, I wouldn't put it here for consistency.

Comment (alexppg) [inline docs/command_line.md:90]: In this doc there's no mention of the additional dependencies that the commands might need, I wouldn't put it here for consistency. 
```suggestion
```

File: docs/command_line.md (hunk 1 above)
Comment (f-arruza) [reply docs/command_line.md]: done

File: docs/command_line.md (hunk 2 above)
Comment (f-arruza) [reply docs/command_line.md]: removed

**Pull request merged**